Service/CreateMarriageService.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace CommonGateway\HuwelijksplannerBundle\Service;
+
+use App\Service\ObjectEntityService;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
+use Symfony\Component\Security\Core\Security;
+
+/**
+ * This service holds al the logic for creating the marriage request object.
+ */
+class CreateMarriageService
+{
+    private EntityManagerInterface $entityManager;
+    private ObjectEntityService $objectEntityService;
```

Comment:
You never use this objectEntityService service in this file?
And you should not because this service is old and depricated

Reply:
^